README.md

```diff
@@ -140,19 +140,17 @@ For a cluster with 2 worker pools, named 'default' and 'secondarypool', follow t
       $ terraform state list | grep ibm_container_vpc_worker_pool
         > module.ocp_base.data.ibm_container_vpc_worker_pool.all_pools["default"]
         > module.ocp_base.data.ibm_container_vpc_worker_pool.all_pools["secondarypool"]
+        > module.ocp_base.ibm_container_vpc_worker_pool.pool["default"]
+        > module.ocp_base.ibm_container_vpc_worker_pool.pool["secondarypool"]
```

Comment:
@Aashiq-J I think your saying that we only need to run the state remove command on the default worker pool right? So to avoid any confusion, can you make that clear? Perhaps update the grep to only return default pool in its output?